pkg/lb/servicelb/backendserver.go

```diff
@@ -0,0 +1,25 @@
+package servicelb
+
+import (
+	"net"
+
+	kubevirtv1 "kubevirt.io/api/core/v1"
+
+	"github.com/harvester/harvester-load-balancer/pkg/lb"
+)
+
+type Server struct {
+	*kubevirtv1.VirtualMachineInstance
+}
+
+var _ lb.BackendServer = &Server{}
+
+func (s *Server) GetAddress() (string, bool) {
+	for _, networkInterface := range s.Status.Interfaces {
```

Comment:
Yes, we only need one IPv4 address.

Reply:
suppose one VM has multi nics and different nics binded to different VLANs with separate IPs, will it be an issue here ?

Reply:
We are unable to prove the network connectivity.

Reply:
Then we need to add this limitation to the documents.

On the other hand, if muliti IPs are found in GetAddress(), return error or print error/warning message, will be more robust.

Reply:
Agree to add the limitation to the document and print hint message, but it's better not return error or print error/warning message because multiple IPs is common in the Harvester.

Reply:
@yaocw2020 please log an issue to update the doc, write down this limitation, thanks.

Reply:
I start to write the document.